lib/howitzer/capybara_helpers.rb

```diff
@@ -130,6 +132,17 @@ def cloud_resource_path(kind)
 
     private
 
+    def apply_user_agent(caps)
```

Comment:
bad idea. This code will be read-only in project. Put the code to appropriate drivers in capybara.rb